snap/snapcraft.yaml

```diff
@@ -37,6 +47,13 @@ apps:
     command: bin/otbr-setup.sh
     daemon: oneshot
     install-mode: disable
+    plugs:
+      - network
+      - network-bind
+      - network-control
+      - firewall-control
+      - system-observe
+      - system-etc-run-files
```

Comment:
From PR description:
> system-files plugs to access system information and files in order to set up iproute (/etc/sysctl.* and /etc/iproute2).

In that case, why is the app also given access to /run? Also, why other apps are given access to /etc?

I suggest splitting `system-etc-run-files` into two plugs so that each app is granted the bare minimum acess.

Reply:
I have divided it into two plugs: `system-config-files` and `system-runtime-files`, granting applications access either `/etc` or `/run` for different purposes.